view/code/index.ts

```diff
@@ -88,6 +88,21 @@ function listenChanges(input: HTMLInputElement): void {
     }
   }
 
+  function processRgbBlur(): void {
+    let value = input.value.trim()
+    let parsed = parseAnything(value)
+
+    if (parsed && parsed.mode in formats.get()) {
+      // Two options in <OutputFormats> — hex and rgb — are parsed as rgb.
+      let newFormat = isHexNotation(parsed.mode) ? 'hex' : parsed.mode
+      if (newFormat !== outputFormat.get()) {
+        outputFormat.set(newFormat as OutputFormats)
```

Comment:
Don’t like this casting. It will be better to have function like `isOutputFormat()` which will set type of `parsed.mode`

Reply:
Added `toOutputFormat`